test/integration/contentful_test.exs

```diff
@@ -21,7 +21,7 @@ defmodule Contentful.DeliveryTest do
     use_cassette "entry" do
       entry = Delivery.entry(@space_id, @access_token, "5JQ715oDQW68k8EiEuKOk8")
 
-      assert is_map(entry["fields"])
+      assert is_list(entry["fields"])
```

Comment:
When I run the test, I get this output for the entry (from a bare contentful blog) :

```
%{"fields" => [{"biography",
    "Mike Springer is a journalist living in Cambridge, Massachusetts, he writes daily for Open Culture."},
   {"createdEntries",
    [%{"sys" => %{"id" => "A96usFSlY4G0W4kwAqswk", "linkType" => "Entry",
         "type" => "Link"}}]}, {"name", "Mike Springer"},
   {"profilePhoto",
    %{"fields" => %{"file" => %{"contentType" => "image/jpeg",
          "details" => %{"image" => %{"height" => 200, "width" => 200},
            "size" => 3559}, "fileName" => "kenchoong.jpeg",
          "url" => "//images.contentful.com/z3aswf9egfi8/2xA3oKlZTuQ0Wgs2Wm2Mkk/f4b8b7a9121dceccb7d69cd41bdadea5/kenchoong.jpeg"},
        "title" => "Mike Springer"},
      "sys" => %{"createdAt" => "2016-06-30T19:34:17.974Z",
        "id" => "2xA3oKlZTuQ0Wgs2Wm2Mkk", "locale" => "en-US", "revision" => 1,
        "space" => %{"sys" => %{"id" => "z3aswf9egfi8", "linkType" => "Space",
            "type" => "Link"}}, "type" => "Asset",
        "updatedAt" => "2016-06-30T19:34:17.974Z"}}},
   {"website", "https://plus.google.com/+openculture/posts"}],
  "sys" => %{"contentType" => %{"sys" => %{"id" => "1kUEViTN4EmGiEaaeC6ouY",
        "linkType" => "ContentType", "type" => "Link"}},
    "createdAt" => "2016-06-30T19:34:24.984Z", "id" => "5JQ715oDQW68k8EiEuKOk8",
    "locale" => "en-US", "revision" => 1,
    "space" => %{"sys" => %{"id" => "z3aswf9egfi8", "linkType" => "Space",
        "type" => "Link"}}, "type" => "Entry",
    "updatedAt" => "2016-06-30T19:34:24.984Z"}}
```

I'm not familiar yet with contentful's API but this does sound fair. What do you think ?